typer/cli.py

```diff
@@ -1,13 +1,16 @@
 import importlib.util
 import re
 import sys
+from collections import defaultdict
+from itertools import chain
 from pathlib import Path
-from typing import Any, List, Optional
+from typing import Any, Dict, List, Optional
 
 import click
 import typer
 import typer.core
 from click import Command, Group, Option
+from typer.rich_utils import _RICH_HELP_PANEL_NAME, COMMANDS_PANEL_TITLE
```

Comment:
There's two issues here:
- this will simply crash when `rich` is not installed
- even if `rich` is installed, we want to [lazy-load](https://github.com/fastapi/typer/pull/1128) it

I will have a look into fixing this.